pom.xml

```diff
@@ -75,14 +75,18 @@
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
                 <artifactId>bom-${jenkins.baseline}.x</artifactId>
-                <version>4948.vcf1d17350668</version>
+                <version>5422.v0fce72a_b_b_8cf</version>
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
         </dependencies>
     </dependencyManagement>
     <dependencies>
         <!-- libraries -->
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>commons-lang3-api</artifactId>
```

Comment:
https://github.com/jenkinsci/pipeline-groovy-lib-plugin/pull/195#pullrequestreview-3202658758

Better I think to revert this part, and I can offer a #195 replacement separately.